includes/Integrations/Core_Themes_Support.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Class Core_Themes_Support
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
```

Comment:
Originally the Integrations namespace was designed for third party plugins / themes. This is an intergration with WordPress itself, feels a little weird for it to live here. Thoughts @swissspidy ?

Reply:
It is kind of an integration we are adding for the codes that we do not govern, hence put it under integration.

Reply:
Seems to be quite a good fit for this namespace IMO 👍